js/index.js

```diff
@@ -0,0 +1,38 @@
+window.onload = function (){
```

Comment:
Add comments also pls

Reply:
@Kuan-Zher Added code documentation for the functions here too also